.happy/terraform/modules/batch/main.tf

```diff
@@ -43,9 +43,9 @@ resource aws_batch_job_definition batch_job_def {
       "value": "${var.frontend_url}"
     }
   ],
-  "vcpus": 2,
+  "vcpus": 8,
```

Comment:
What machines are we using in each environment? Ideally, since we allocate a static amount of memory, this should be proportional to the available vCPUs.